persistence/postgres-persistence/build.gradle

```diff
@@ -1,5 +1,8 @@
 dependencies {
 
+    compileOnly "org.projectlombok:lombok:${revLombok}"
```

Comment:
why do we need lombok ?

Reply:
If we need to use lombok, have a look at 

https://projectlombok.org/features/experimental/Delegate

to use in the OutboxExecutionDao delegation

Reply:
Found out lombok not needed at all. Logger used for logging.